docs/src/examples/components/Slider/Types/SliderExample.shorthand.steps.ts

```diff
@@ -0,0 +1,32 @@
+import { Slider } from '@stardust-ui/react/src'
+
+const selectors = {
+  input: `.${Slider.slotClassNames.input}`,
+}
+
+const focusSliderStep: ScreenerStep = (builder, keys) => builder.keys('body', keys.tab)
+
+const config: ScreenerTestsConfig = {
```

Comment:
Let's make sure that we capture this for the rtl example as well

Reply:
good idea, done